srv/external.cds

```diff
@@ -0,0 +1,46 @@
+using { API_BUSINESS_PARTNER as external } from './external/API_BUSINESS_PARTNER';
+
+/**
+ * Tailor the imported API to our needs...
+ */
+extend service external with {
+
+  /**
+   * Simplified view on external addresses
+   */
+  entity Addresses as projection on external.A_BusinessPartnerAddress {
+    key AddressID,
+    key BusinessPartner,
+    @readonly Country,
+    @readonly CityName,
+    @readonly PostalCode,
+    @readonly StreetName,
+    @readonly HouseNumber
+  }
+
+  /**
+   * Add asynchronous eventing API
+   */
+  event BO_BusinessPartner_Changed {
+    ![KEY]: Array of {
```

Comment:
How does this `![KEY]` notation work?

Reply:
the `![]` notation is the cds notation for quoting. As `key` is a cds keyword it needs to be quoted to be used as an element name here.

Reply:
I didn't really find this documented on capire, but there are some hints scattered across release notes: https://github.wdf.sap.corp/pages/cap/releases/nov20#quickfixes-for-deprecated-identifiers. Also writing it differently (e.g. using double quotes) creates a compiler warning.